index.js

```diff
@@ -1,13 +1,14 @@
 /* eslint-env node */
-'use strict';
+"use strict";
 
 // if using native backstop remote server
-const BACKSTOP_PROXY_PATH = '/backstop';
-const BACKSTOP_PROXY_TARGET = 'http://localhost:3000';
```

Comment:
All explicit references to port `3000` have been replaced with `BACKSTOP_REMOTE_HTTP_PORT` - however they still fallback to `3000`, so the current/default behaviour is unchanged